keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -243,147 +242,161 @@ A `Bucket` is not deleted if it is bound to a `BucketRequest`.
+An immutable, cluster-scoped, custom resource providing a way for admins to specify policies used to access buckets. A `BucketAccessClass` (BAC) is referenced by a user [BucketAccessRequest](#bucketaccessrequest) and is used to populate the [BucketAccess](#bucketaccess) instance. A BAC references a config map which is expected to define the access policy for a given provider. It is typical for these config maps to reside in each provisioner's namespace, though this is not required. Unlike a [BucketClass](#bucketclass), there is no default BAC.
 
 ```yaml
-apiVersion: cosi.io/v1alpha1
+apiVersion: objectstorage.k8s.io/v1alpha1
 kind: BucketAccessClass
 metadata: 
   name:
-provisioner: [1]
-policyActionsConfigMap: [2]
+policyActionsConfigMap: [1]
   name:
   namespace:
-parameters: [3]
+parameters: [2]
 ```
 
-1. `provisioner`: (optional) the name of the driver that `BucketAccess` instances should be managed by. If empty then there is no driver/sidecar, thus the admin had to create the `Bucket` and `BucketAccess` instances.
 1. `policyActionsConfigMap`: (required) a reference to a ConfigMap that contains a set of provisioner/platform-defined policy actions for bucket access. It is seen as typical that this config map's namespace is the same as for the provisioner. In othe words, a namespace that has locked down RBAC rules or prevent modification of this config map.
-1. `parameters`: (Optional)  A map of string:string key values.  Allows admins to control user and access provisioning by setting provisioner key-values. Optional reserved keys cosi.io/configMap and cosi.io/secrets are used to reference user created resources with provider specific access policies.
+1. `parameters`: (optional)  A map of string:string key values.  Allows admins to control user and access provisioning by setting provisioner key-values. Optional reserved keys objectstorage.k8s.io/configMap and objectstorage.k8s.io/secrets are used to reference user created resources with provider specific access policies.
 ---
 
 ### App Pod
-The application pod utilizes CSI's inline ephemeral volume support to provide the endpoint and secret credentials in an in-memory volume. This approach also, importantly, prevents the pod from launching before the bucket has been provisioned since the kubelet waits to start the pod until it has received the cosi-node-adapter's _NodeStageVolume_ response.
+The application pod utilizes CSI's inline ephemeral volume support to provide the endpoint and secret credentials in an in-memory (ephemeral) volume. This approach also, importantly, prevents the pod from launching before the bucket has been provisioned since the kubelet waits to start the pod until it has received the cosi-node-adapter's _NodeStageVolume_ response.
```

Comment:
We need a mechanism to request a bucket while provisioning a pod, just like claims are requested when provisioning pods. Since it is impossible to know the secret name upfront, while provisioning the pod, we need a CSI driver that will do it

Reply:
dup of above thread re secrets